### PR TITLE
Cleaning up the graph

### PR DESCRIPTION
**Pull request by dellaert**
Made all small things about images concrete rather than delayed. This has a huge effect on the dask graph, which now looks much smaller/tighter:

image

Only image retrieval is still delayed.

There are still many `getitem` tasks that I don't get (no pun intended) but that's for a later PR.

### CODE REVIEW

File: gtsfm/bundle/bundle_adjustment.py [hunk 1]
```diff
@@ -395,7 +395,7 @@ def create_computation_graph(
         sfm_data_graph: Delayed,
         absolute_pose_priors: List[Optional[PosePrior]],
         relative_pose_priors: Dict[Tuple[int, int], PosePrior],
-        gt_cameras_graph: Optional[List[Delayed]] = None,
+        gt_cameras: Optional[List[Delayed]] = None,
```
Comment (ayushbaid) [inline gtsfm/bundle/bundle_adjustment.py:398]: We can make it `Optional[List[gtsfm_types.CAMERA_TYPE]]`

File: gtsfm/frontend/verifier/verifier_base.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def create_computation_graph(
             image_pair_indices: 2-tuple (i1,i2) specifying image pair indices
             detection_graph: nodes with features for each image.
             matcher_graph: nodes with matching results for pairs of images.
-            camera_intrinsics_graph: nodes with intrinsics for each image.
+            all_intrinsics: nodes with intrinsics for each image.
```
Comment (ayushbaid) [inline gtsfm/frontend/verifier/verifier_base.py:89]: the variables in the arg don't match the docstring.

File: gtsfm/runner/gtsfm_runner_base.py [hunk 3]
```diff
@@ -129,12 +129,12 @@ def run(self) -> None:
             num_images=len(self.loader),
             image_pair_indices=image_pair_indices,
             image_graph=self.loader.create_computation_graph_for_images(),
-            camera_intrinsics_graph=self.loader.create_computation_graph_for_intrinsics(),
-            image_shape_graph=self.loader.create_computation_graph_for_image_shapes(),
+            all_intrinsics=self.loader.get_all_intrinsics(),
+            image_shape_graph=self.loader.get_image_shapes(),
```
Comment (ayushbaid) [inline gtsfm/runner/gtsfm_runner_base.py:133]: Change the arg to image_shapes in scene optimizer?

File: gtsfm/scene_optimizer.py [hunk 4]
```diff
@@ -112,12 +113,12 @@ def create_computation_graph(
         num_images: int,
         image_pair_indices: List[Tuple[int, int]],
         image_graph: List[Delayed],
-        camera_intrinsics_graph: List[Delayed],
+        all_intrinsics: List[Delayed],
```
Comment (ayushbaid) [inline gtsfm/scene_optimizer.py:116]: Should not be delayed

**Review by ayushbaid — APPROVED** [review]
LGTM. Does it also make the scheduling faster? I have to wait a long time for tasks to start executing on my local machine using the current master.

**Comment by dellaert** [comment]
> LGTM. Does it also make the scheduling faster? I have to wait a long time for tasks to start executing on my local machine using the current master.

Yes, it feels faster at startup.

**Pull request merged**